examples/preprocessing/plot_find_ref_artifacts.py

```diff
@@ -0,0 +1,116 @@
+"""
+.. _ex-megnoise_processing:
+
+==================
+Find MEG reference channel artefacts
+==================
+
+Use ICA decompositions of MEG reference channels to remove intermittent noise.
+
+Many MEG systems have an array of reference channels which are used to remove
+external magnetic noise. However, standard removal techniques often fail when
+noise is intermittent. This technique often succeeds where the standard
+techniques do not.
+
+There are two algorithms to choose from: separate and together (default). In
+the "separate" algorithm, two ICA decompositions are made: one on the reference
+channels, and one on reference + standard channels. The reference + standard
+channel components which correlate with the reference channel components are
+removed.
+
+In the "together" algorithm, a single ICA decomposition is made on reference +
+standard channels, and those components whose weights are particularly heavy
+on the reference channels are removed.
+
+This technique is fully described and validated in :footcite:`HannaEtAl2020`
```

Comment:
You need a corresponding
```
##########################################################
# References
# ----------
# .. footbibliography::
```
or so at the bottom of the file, see how we use `footbibliography` in other examples